projects/storefrontapp-cds-integration/src/app/app.module.ts

```diff
@@ -83,6 +83,7 @@ if (!environment.production) {
             'http://tag.static.stage.context.cloud.sap/js/profile-tag.js',
           configUrl:
             'https://tag.static.stage.context.cloud.sap/config/dfbb97b0-f4d7-11e9-9c99-2125ab7968c6',
+          allowInsecureCookies: true,
```

Comment:
this shouldn't be here (for testing purposes locally)

Reply:
good to know. So the tests in the pipeline run in ssl mode?

Reply:
not necessarily, it builds for production and runs on a server.

Reply:
ok, if it doesn't run ssl then this 'allowInsecureCookies' will be required

Reply:
it can stay